src/ocamlorg_frontend/components/learn_components.eml

```diff
@@ -154,31 +154,46 @@ let right_books_content_block books ~title ~(link : link) =
+      <ul class="list-disc marker:text-orange-500">
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+      </ul>
     </div>
-    <div class="mt-4 lg:mt-0 lg:col-span-1 lg:col-start-3">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Still need help?</h2>
-      <div class="text-sm mb-2">
-        <a href="https://discuss.ocaml.org/" class="font-semibold no-underline bg-transparent cursor-pointer hover:underline">
-          <%s! Icons.community "inline h-4 w-4" %>
-          Ask the OCaml Community</a
-        >
+    <div class="grid grid-cols-2 gap-4 border-t border-gray-200 px-6">
+      <div class="border-r border-gray-200 pb-3">
+        <h2 class="mb-2 text-lg font-bold leading-6">Still need help?</h2>
+        <div class="mb-2 text-sm flex items-center gap-2">
+          <a href="https://discuss.ocaml.org/" class="font-normal text-lg no-underline bg-transparent cursor-pointer hover:underline">
+            <%s! Icons.user_group "inline h-5 w-5" %>
+            Ask the OCaml Community</a
+          >
+        </div>
+        <div class="mb-2 text-sm flex items-center gap-2">
+          <a href="https://github.com/ocaml/ocaml.org/issues/new" class="font-normal text-lg no-underline bg-transparent cursor-pointer hover:underline">
+            <%s! Icons.github_issue "inline h-4 w-4" %>
+            Open an Issue</a
+          >
+        </div>
       </div>
-      <div class="text-sm mb-2">
-        <a href="https://github.com/ocaml/ocaml.org/issues/new" class="font-semibold no-underline bg-transparent cursor-pointer hover:underline">
-          <%s! Icons.github_issue "inline h-4 w-4" %>
-          Open an Issue</a
-        >
+      <div class="pb-3">
+        <h2 class="mb-2 text-lg font-bold leading-6">Help Improve Our Documentation</h2>
+        <p class="text-sm text-lighter leading-relaxed mb-0">All OCaml docs are open source. See something that's wrong and unclear?</p>
+        <div class="mb-5 text-sm flex items-center gap-2">
+          <a href="<%s href %>" class="cursor-pointer bg-transparent font-normal text-lg no-underline hover:underline">
+            <%s! Icons.raised_hand "inline h-6 w-6" %>
+            Contribute</a
+          >
+        </div>
```

Comment:
```suggestion
        <div class="mb-5 text-sm">
          <a href="<%s href %>" class="flex items-center gap-2 bg-transparent font-normal text-lg no-underline hover:underline">
            <%s! Icons.raised_hand "h-6 w-6" %>
            Contribute</a
          >
        </div>
```

To make the icon and text within the `a` element have the gap, the flex styles need to go on the `a` element.

Same is true for the other links.

I think one of the other two needs a `gap-3` (because one time, we have an icon with `h-5 w-5` and the other time we have a `h-4 w-4` icon). Then, the text will align beautifully.

Great work :+1: